### ISSUE TITLE
webview rendering of sphinx broken

### ISSUE THREAD

**Issue by xrayn** [vscode-restructuredtext/vscode-restructuredtext#126]
**Environment**
- [x ] Extension version: 84.0.0
- [x ] Visual Studio Code version: 1.29.1
- [x ] Operating System version: Windows 10
- [x ] Python version: 3.7
- [ ] sphinx version:
- [ ] A sample of conf.py or its important fragments:

**Expected behavior**
rendering of sphinx's html output does not look weird

**Actual behavior**

rendering of tables and navigation pane broken

See screenshot.
Left side Navigation/Search Pane and table broken.
Right side correctly rendered by chrome.

![image](https://user-images.githubusercontent.com/1036978/49692336-641b3a80-fb58-11e8-9941-beaa5e6117df.png)

Identified problem, `basic.css` file from `classic.css` and `alabaster.css`
   @import url("basic.css");

is not loaded properly.

Creating a file `_static/custom.css` with contents as equally defined in `basic.css` fixes the table rendering:

    table.docutils {
       border: 0;
       border-collapse: collapse ;
    }

**Comment by lextm** [comment]
Can you share a test project? You can send them to my mailbox support@lextm.com if you don't want to share it on GitHub directly.

81.0.0 switched to WebView, which does have strict restrictions on what can be loaded. So with the test project I can see how the links should be patched.

**Comment by xrayn** [comment]
Hi, sharing is no problem.

[stest.zip](https://github.com/vscode-restructuredtext/vscode-restructuredtext/files/2660366/stest.zip)

The project was created like this:

```
mkdir stest
cd stest
sphinx-quickstart # select all default options
code .
```

the content of index.rst is:

```
.. testrender documentation master file, created by
   sphinx-quickstart on Sun Dec  9 02:05:28 2018.
   You can adapt this file completely to your liking, but it should at least
   contain the root `toctree` directive.

Welcome to testrender's documentation!
======================================

.. toctree::
   :maxdepth: 2
   :caption: Contents:



Indices and tables
==================

* :ref:`genindex`
* :ref:`modindex`
* :ref:`search`


+-------+-------+---+----+----+
|A      | B     | C | D  | E  |
+=======+=======+===+====+====+
|1      | 3     | 4 | 2  | 6  |
+-------+-------+---+----+----+
|1      | 3     | 4 | 2  | 6  |
+-------+-------+---+----+----+
|1      | 3     | 4 | 2  | 6  |
+-------+-------+---+----+----+
|1      | 3     | 4 | 2  | 6  |
+-------+-------+---+----+----+
|1      | 3     | 4 | 2  | 6  |
+-------+-------+---+----+----+

```

**Comment by lextm** [comment]
Just published 85.0.0. It removes the security restrictions and the style file loading should work as expected now.

**Comment by rizickus** [comment]
That kind of problem still exists on my setup using 85.0.0 version. If this fix works for @xrayn , I will open a separate issue ticket. Guess it can be related to the issue #124 .

**Comment by xrayn** [comment]
Problem still exists. `basic.css` is still not loaded implicitly.

**Comment by xrayn** [comment]
FYI: In Linux the sphinx rendering works as expected with version 85.0.0. Windows still broken.

Version: 1.29.1
Commit: bc24f98b5f70467bc689abf41cc5550ca637088e
Date: 2018-11-15T19:07:43.495Z
Electron: 2.0.12
Chrome: 61.0.3163.100
Node.js: 8.9.3
V8: 6.1.534.41
Architecture: x64

**Comment by lextm** [comment]
Thanks for the extra info. I will try to find a Windows machine soon to reproduce the issues there.

**Comment by lextm** [comment]
Fixed #127. Please test release 86.0.0, and it should now work on Windows properly.

**Comment by xrayn** [comment]
works beautifully, thanks!
